libs/wingc/src/lib.rs

```diff
@@ -48,6 +48,50 @@ pub struct CompilerOutput {
 	pub diagnostics: Diagnostics,
 }
 
+// Hack needed for some WASM related reason
```

Comment:
Any additional info on this "hack"?

Reply:
I can try to put some more details in the comment, but some of it is a guess. I'll see what I can do/lookup.

Reply:
Thanks for bringing this up, because I looked into it more and saw I was doing the wrong thing. For WASI libraries, it's actually looking for an `_initialize` function to be exposed rather than _start. 
When creating a `cdylib` like we are here, an _initialize function is automatically added so I just deleted this function.